### DISCUSSION TITLE
How can I disable new line behavior?

### DISCUSSION THREAD

**Discussion by eroluysal** [prettier/prettier#14114] [Q&A]
Hey, 

How can I disable this behavior of prettier?

**Expected**

    <span className='hello-world-inner' data-bs-toggle='workflow'>Queue Workers</span>

**Output**

    <span className='hello-world-inner' data-bs-toggle='workflow'>
      Queue Workers
    </span>

Is it possible to remove newlines?

Thanks.

**Comment by Nadeera3784** [comment]
Hi there @eroluysal,
Try to add this to your config file 
```
{
 "jsxBracketSameLine": true
}
```

**Reply by eroluysal** [reply]
Hi @Nadeera3784 

unfortunately it didn't work
